packages/integration-tests/src/playbook/hardhat.config.ts

```diff
@@ -1,13 +1,21 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-require("dotenv").config({ path: __dirname + "/.env" });
-
 import "@matterlabs/hardhat-zksync-deploy";
 import "@matterlabs/hardhat-zksync-solc";
 import "@nomiclabs/hardhat-ethers";
+import * as fs from "fs";
+
 import { localConfig } from "../config";
+import { Wallets } from "../entities";
 
 import type { HardhatUserConfig } from "hardhat/types";
 
+const envFilePath = `${__dirname}/.env`;
```

Comment:
We shouldn't do any file existence checks. We should just have a fallback to `Wallets.richWalletPrivateKey`.  No logic is needed here in this file except loading dotenv.

Reply:
@Romsters updated